### PR TITLE
Add parameter to creation to support authSchema

### PR DESCRIPTION
**Pull request by malixsys**
add a `--oldVersion` parameter to `mongo:create` to set `authSchema.currentVersion` to `3`

@josegonzalez Links to issue #39

### CODE REVIEW

**Comment by josegonzalez** [comment]
What exactly does this authschema thing do?

**Comment by malixsys** [comment]
It allows for backward compatibility with MONGODB-CR credentials, which many backups and programs use (like Robomongo)

https://docs.mongodb.org/v3.0/release-notes/3.0-scram/#considerations

File: commands [hunk 1]
```diff
@@ -50,6 +50,12 @@ case "$1" in
     dokku_log_verbose_quiet "Waiting for container to be ready"
     docker run --rm --link "$SERVICE_NAME:$PLUGIN_COMMAND_PREFIX" dokkupaas/wait > /dev/null
 
+    if [[ "$3" == "--oldSchema" ]]; then
```
Comment (josegonzalez) [inline commands]: Can you iterate over `$@` instead of checking a specific param?

Comment (malixsys) [inline commands]: My bash-fu is not strong enough...

Comment (josegonzalez) [inline commands]: My merge-fu is not strong enough... ;)

Helpful stackoverflow post: https://stackoverflow.com/questions/255898/how-to-iterate-over-arguments-in-bash-script

**Comment by malixsys** [comment]
Workaround found. Closing unattended PR's

**Pull request closed without merging**